Projects/Addons/Tests/ServiceTests/AddonsServiceTests.swift

```diff
@@ -0,0 +1,96 @@
+@preconcurrency import XCTest
+import hCore
+
+@testable import Addons
+
+@MainActor
+final class AddonsServiceTests: XCTestCase {
+    weak var sut: MockAddonsService?
+
+    override func setUp() async throws {
+        try await super.setUp()
+        sut = nil
+    }
+
+    override func tearDown() async throws {
+        Dependencies.shared.remove(for: AddonsClient.self)
+        try await Task.sleep(nanoseconds: 100)
+
+        XCTAssertNil(sut)
+    }
+
+    func testFetchAddonDataSuccess() async {
```

Comment:
`    func testFetchAddonDataSuccess() async {
`
to 
`    func testFetchAddonDataSuccess() async throws {
`
and than you can replace try! with try